packages/webapp/src/routes/registration-submitted.js

```diff
@@ -1,9 +1,8 @@
 import constants from '../utils/constants.js'
-import { formatAppRef } from '../utils/helpers.js'
 
 const handlers = {
   get: async (request, h) => {
-    const applicationReference = formatAppRef(request.yar.get(constants.redisKeys.APPLICATION_REFERENCE))
+    const applicationReference = request.yar.get(constants.redisKeys.APPLICATION_REFERENCE)
```

Comment:
If formatAppRef() is not required as per new implementation, have to remove it from the helper and also need to update all the references for this function (Eg:  from webapp/src/routes/registration-submitted.js)